src/Chainweb/Pact/RestAPI.hs

```diff
@@ -119,6 +127,22 @@ pactPollApi
     . Proxy (PactPollApi v c)
 pactPollApi = Proxy
 
+-- -------------------------------------------------------------------------- --
+-- POST Queries for Pact Local Pre-flight
```

Comment:
Is the comment too specific? (Is this POST query also for /local endpoint with no pre-flight?)

Reply:
It's only for the preflight api - the original local api definition comes from Pact proper

Reply:
Ah. I think I was conflating "preflight API" with a particular preflight query parame.